addons/mail/static/src/core_ui/message.xml

```diff
@@ -20,6 +20,7 @@
                 t-on-click="onClick"
                 t-on-mouseenter="onMouseenter"
                 t-on-mouseleave="onMouseleave"
+                t-on-contextmenu="(ev) => reactionViewer.open(ev)"
```

Comment:
I think this spec is outdated.
Instead of long pressing / right-click, we should have a message option to "View reactions".
Maybe `t-on-contextmenu` still ok on a reaction group, but certainly not on the message.

To talk to our PO whether agree or not.